### DISCUSSION TITLE
Student development pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#35882] [GitHub Education]
I want a student development pack.but i. Can't get even know also .what can I do now .please help me to get the student developement pack.

**Comment by Neodevils** [comment]
Hello, @Balusadda123 ! 👋🏻

You have to prove that you're a student with a school ID card or diploma. You will get reply from GitHub soon. After that, you can get your student packs for just going through for the websites offering you **GitHub Student Pack** such as Figma, Azure DevOps etc.

**Reply by ghost** [reply]
Please send me the student developement pack

**Reply by Neodevils** [reply]
You have to register for it.
